CRM/Admin/Form/Preferences/Display.php

```diff
@@ -105,6 +106,8 @@ public function postProcess() {
 
     $this->postProcessCommon();
 
+    \Civi::service('asset_builder')->clear();
```

Comment:
Generally, this is a nice use of the asset-builder. 👍 

I think we could make this work without a cache-clear -- if `crm-menubar.css` accepted the `menubar_color` as a param, e.g.

```php
$items[] = Civi::service('asset_builder')->getUrl('crm-menubar.css', [
  'color' => Civi::settings()->get('menubar_color'),
]);
```

The asset-builder tracks the params as part of its cache logic -- so whenever you use a different param, you don't need to clear anything.  It just builds a new variant with the given value.

Here's a sub PR: https://github.com/colemanw/civicrm-core/pull/8